prov/efa/src/efa.h

```diff
@@ -89,7 +89,11 @@
 #define EFA_MR_IOV_LIMIT 1
 #define EFA_MR_SUPPORTED_PERMISSIONS (FI_SEND | FI_RECV)
 
-#define EFA_DEF_NUM_MR_CACHE 36
+/*
+ * Multiplier to give some room in the device memory registration limits
```

Comment:
This multiplier is really only applicable if there is only one domain that is opened. Otherwise, with two domains, we would be sitting at 180% of the device limits as our limit, which would prevent additional processes added if they have already fully utilized the device limits...assuming that multiple domains are opened on the same device. This may be a bit misleading, but no real requested change.

Reply:
Right, this only helps when a single process is opened, some registrations happen, and then another process is added. If multiple processes are in the mix and consume all of the resources then the user will have to rely on the failure to register bounce buffers to fix their app or set ulimits.